caffe2/python/caffe_translator.py

```diff
@@ -92,36 +92,50 @@ def TranslateModel(
                 'only accepts new style layers that are stored in the '
                 'layer field.'
             )
-        for layer in caffe_net.layer:
-            if not _ShouldInclude(net_state, layer):
+        i = 0
+        pretrained_layers = [l for l in pretrained_net.layer] +\
+                            [l for l in pretrained_net.layers]
+        while i < len(caffe_net.layer):
```

Comment:
```for i in range(0, len(caffe_net.layer))``` perhaps is better. I don't got why original ``` for layer in caffe_net.layer:``` wasn't good enough for this iteration.